config/initializers/slack_notifier.rb

```diff
@@ -1,7 +1,9 @@
 class NoOpHTTPClient
+  # rubocop:disable Style/OptionHash
   def self.post(uri, params = {})
```

Comment:
**nitpick (non-blocking):** Would using `**kwargs` work here instead of `params = {}`? Then we wouldn't have to disable the cop.